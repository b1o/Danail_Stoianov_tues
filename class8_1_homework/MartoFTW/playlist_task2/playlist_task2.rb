require "csv"

out_arr = []

	CSV.foreach(ARGV[0]) do |row|

		if row[1].to_s == ARGV[1].to_s

			if row[3].to_f < 4.00
		
				out_arr << row

			end

		end

	end

	out_arr = out_arr.sort_by{|a,b,c,d| d}
	out_arr.reverse!

	CSV.open("playlist_task2_result.csv","w") do |csv|
		
		out_arr.each do |elem|

			csv << elem

		end
	
	end
