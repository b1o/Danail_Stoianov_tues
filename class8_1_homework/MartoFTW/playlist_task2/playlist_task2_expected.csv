Umri,Azis,Popfolk,3.51
